src/planning.py

```diff
@@ -423,6 +424,65 @@ def _run_low_level_search(task: Task, option_model: _OptionModelBase,
     return [], True
 
 
+def _run_plan_with_option_model(
```

Comment:
in all these calls to LowLevelTrajectory i noticed that you're using `is_demo=False` -- why is this okay? aren't we calling this function within `demo_only.py` which is supposed to generate demos? i feel like a lot of our learning code is set up to ignore non-demos, so this might cause issues at learning time. actually in light of this, i'm confused how any operators are being learned...

Reply:
Good catch! This line saved us: https://github.com/Learning-and-Intelligent-Systems/predicators/blob/behavior-simulate-option-model/src/datasets/demo_only.py#L239